.vscode/settings.json

```diff
@@ -10,5 +10,14 @@
         "**/node_modules": true,
         "dist/": true
     },
-    "editor.tabSize": 2
+    "editor.tabSize": 2,
+    "editor.formatOnSave": true,
+    "prettier.singleQuote": false,
+    "prettier.trailingComma": "es5",
+    "prettier.typescriptEnable": [
+        "typescript",
+        "typescriptreact"
+    ],
+    "prettier.semi": false,
+    "prettier.printWidth": 120
```

Comment:
On https://github.com/prettier/prettier#options they emphatically recommend that users stick to `80` since Prettier (very unfortunately, IMO) has no concept of vertical space until it hits the width boundary. E.g.,

```javascript
  const {
    foo: {
      bar: {
        baz
      }
    },
    foo: {
      bar: {
        baz
      }
    },
    foo: {
      bar: {
        baz
      }
    },
    foo: {
      bar: {
        baz
      }
    },
  } = props
```
is collapsed into 
```javascript
  const { foo: { bar: { baz } }, foo: { bar: { baz } }, foo: { bar: { baz } }, foo: { bar: { baz } } } = props
```
(110 chars long)

Since we're dealing with React and propTypes where this sort of thing is common, this can get very unreadable very fast. I really want to argue keeping this value at 80 or max 90 if possible...